doc/sphinxman/document_psivariables.pl

```diff
@@ -161,39 +163,75 @@ sub get_description
                     foreach my $line (@text) {
                         if ($file =~ /\.cc\z/) {
                             if ($line =~ /\QProcess::environment.globals\E/) {
-                                my @ltemp = split( /"/, $line);
-                                if ($ltemp[0] =~ /\QProcess::environment.globals\E/) {
-                                    push(@EnvVariables, $ltemp[1]);
+                                if ($line =~ /\Qno-autodoc\E/) {
```

Comment:
A trip down memory lane!

Reply:
yeah, I was shocked to find one can't even print out an array in perl w/o importing a helper fn.